types/model/name.go

```diff
@@ -686,7 +686,10 @@ func IsValidNamePart(kind PartKind, s string) bool {
 		return false
```

Comment:
For line 682: should be limit of 80 for a username, model or tag name

Reply:
It should also check the size is at least 2 i.e.

```
if len(s) < 2 {
  return false
}
``

Reply:
What about hosts as per: https://github.com/ollama/ollama/pull/3887#issuecomment-2075881949

Reply:
Host can be <2 (1 char?) I guess

Reply:
Correction on tags: 1 character should be ok there too - sorry